README.md

```diff
@@ -127,37 +122,27 @@ Everything in this repository is BSD style license unless otherwise specified.
 
 Copyright (c) 2015 The Polymer Authors. All rights reserved.
 
-## Helper utilities
-
-### `WebComponentsReady`
-
-Under native HTML Imports, `<script>` tags in the main document block the loading of such imports. This is to ensure the imports have loaded and any registered elements in them have been upgraded.
+## Changes in version 2.x
 
-The `webcomponents-lite.js` polyfill (as well as the smaller bundles and the loader) parse element definitions and handle their upgrade asynchronously. If prematurely fetching the element from the DOM before it has an opportunity to upgrade, you'll be working with an `HTMLUnknownElement`.
-
-For these situations, you can use the `WebComponentsReady` event as a signal before interacting with the element. The criteria for this event to fire is all Custom Elements with definitions registered by the time HTML Imports available at load time have loaded have upgraded.
-
-```js
-window.addEventListener('WebComponentsReady', function(e) {
-  // imports are loaded and elements have been registered
-  console.log('Components are ready');
-});
-```
+* The HTML Imports polyfill has been removed. Given that ES modules have shipped in
+most browsers, the expectation is that web components code will be loaded via
+ES modules.
+* When using `webcomponents-loader.js`, scripts that rely on the polyfills *must* be loaded using `WebComponents.waitFor(loadCallback)`.
```

Comment:
Add node that this is only requited if `defer` is used on the script.

Reply:
Done